prereise/gather/griddata/hifld/const.py

```diff
@@ -194,6 +194,79 @@
     "Flywheels",
 }
 
+balancingauthority2interconnect = {
+    "AEC": "Eastern",
+    "AECI": "Eastern",
+    "AVA": "Western",
+    "AVRN": "Western",
+    "AZPS": "Western",
+    "BANC": "Western",
+    "BPAT": "Western",
+    "CHPD": "Western",
+    "CISO": "Western",
+    "CPLE": "Eastern",
+    "CPLW": "Eastern",
+    "CSTO": "Western",
+    "DEAA": "Western",
+    "DOPD": "Western",
+    "DUK": "Eastern",
+    "EEI": "Eastern",
+    "EPE": "Western",
+    "ERCO": "ERCOT",
+    "FMPP": "Eastern",
+    "FPC": "Eastern",
+    "FPL": "Eastern",
+    "GCPD": "Western",
+    "GRIF": "Western",
+    "GRIS": "Western",
+    "GRMA": "Western",
+    "GVL": "Eastern",
+    "GWA": "Western",
+    "HGMA": "Western",
+    "HST": "Eastern",
+    "IID": "Western",
+    "IPCO": "Western",
+    "ISNE": "Eastern",
+    "JEA": "Eastern",
+    "LDWP": "Western",
+    "LGEE": "Eastern",
+    "MISO": "Eastern",
+    "NBSO": "Eastern",
+    "NEVP": "Western",
+    "NSB": "Eastern",
+    "NWMT": "Western",
+    "NYIS": "Eastern",
+    "OVEC": "Eastern",
+    "PACE": "Western",
+    "PACW": "Western",
+    "PGE": "Western",
+    "PJM": "Eastern",
+    "PNM": "Western",
+    "PSCO": "Western",
+    "PSEI": "Western",
+    "SC": "Eastern",
+    "SCEG": "Eastern",
+    "SCL": "Western",
+    "SEC": "Eastern",
+    "SEPA": "Eastern",
+    "SOCO": "Eastern",
+    "SPA": "Eastern",
+    "SRP": "Western",
+    "SWPP": "Eastern",
+    "TAL": "Eastern",
+    "TEC": "Eastern",
+    "TEPC": "Western",
+    "TIDC": "Western",
+    "TPWR": "Western",
+    "TVA": "Eastern",
+    # "WACM": "Western",  # can be Western or Eastern
+    # "WALC": "Western",  # can be Western or Eastern
+    # "WAUW": "Western",  # can be Western or Eastern
```

Comment:
So those are not considered?

Reply:
They are not. The comments are here as an indication that we know the WAPA balancing authorities are missing, and it's an intentional choice.

Reply:
Sounds good